### ISSUE TITLE
UIApplication: AccessibilityLanguage API appears to be missing

### ISSUE THREAD

**Issue by Felix-Dev** [xamarin/xamarin-macios#21925]
### Apple platform

iOS

### Framework version

net8.0-*

### Affected platform version

.NET 8.0.303

### Description

We are currently working on adding accessibility to our app and support for Apple VoiceOver. The display language of our app might differ from the user's system language. For example, the user might have configured their iPhone to display system UI in English, however, our app UI content in German. We've noticed that in this case VoiceOver pronounces our German labels as if they represent English words.

It is possible to configure the language used when speaking the label/accessibility description/etc... using the API
```
accessibilityLanguage
```
(https://developer.apple.com/documentation/objectivec/nsobject/1615192-accessibilitylanguage?language=objc)
While this property can bet set on every _UIView_ it looks like it is also possible to set it globally using the _UIApplication_ instance:
```
application.accessibilityLanguage = @"de";
```
(https://stackoverflow.com/questions/21283862/uiaccessibility-default-language-throughout-the-app)

I have tried to do the same in a MAUI app but I did not find such a property:
![image](https://github.com/user-attachments/assets/f198e182-fddd-40f5-b029-3ab2a29bc2f5)

It looks like the API is missing from the Xamari iOS bindings?

### Steps to Reproduce

n/a

### Did you find any workaround?

No feasible workaround yet.

**Comment by rolfbjarne** [comment]
Looks like our `UIApplication` bindings are missing conformance to the `UIAccessibility` protocol.

I'll have a look.

As a workaround, something like this should work:

```cs
using System.Runtime.InteropServices;
using CoreFoundation;
using Foundation;
using ObjCRuntime;

public static class UIAppearanceExtensions {
	public static void SetAccessibilityLanguage (this NSObject object, string language)
	{
		var languageHandle = CFString.CreateNative (language);
		objc_msgSend (object.GetHandle (), Selector.GetHandle ("setAccessibilityLanguage:"), languageHandle);
		CFString.ReleaseNative (languageHandle);
	}

	[DllImport (Constants.ObjectiveCLibrary)]
	static extern void objc_msgSend (IntPtr handle, IntPtr selector, IntPtr arg0);
}
```

and then you can use it like this:

```cs
application.SetAccessibilityLanguage ("de");
```

**Comment by Felix-Dev** [comment]
@rolfbjarne Thanks for the quick workaround (I can confirm it works) and fix!
